rust/agama-lib/share/profile.schema.json

```diff
@@ -347,7 +351,7 @@
                   "type": "string"
                 }
               },
-              "dns_searchlist": {
+              "dnsSearchlist": {
```

Comment:
I am not sure if openQA testing it, but we should notify them about those backward incompatible changes.